### PR TITLE
Add option to disable caching in setup action

### PR DESCRIPTION
**Pull request by drasmuss**

### CODE REVIEW

File: actions/setup/action.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ inputs:
     description: Python version (passed to actions/setup-python)
     required: false
     default: "3"
+  cache:
+    description: Enable/disable caching of pip/conda downloads
+    required: false
+    default: "true"
```
Comment (tbekolay) [inline actions/setup/action.yml:15]: My gut reaction is to make this of type `boolean` and remove the quotes, but from looking at `mamba-org/setup-micromamba` it seems they don't set a type so I would assume that defaults to string.

**Review by tbekolay — APPROVED** [review]
LGTM!

File: actions/setup/action.yml (hunk 1 above)
Comment (drasmuss) [reply actions/setup/action.yml:15]: That's what I thought at first, but turns out you can't actually have boolean input types for composite actions (see https://github.com/actions/runner/issues/2238).

Comment (tbekolay) [reply actions/setup/action.yml:15]: 😮

**Pull request merged**